aiida_common_workflows/workflows/relax/vasp/generator.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+"""Implementation of `aiida_common_workflows.common.relax.generator.RelaxInputGenerator` for VASP."""
+from aiida import orm
+
+from ..generator import RelaxInputsGenerator, RelaxType
+from .workchain import CommonVASPRelaxWorkChain
+
+__all__ = ('VASPRelaxInputsGenerator',)
+
+
+class VASPRelaxInputsGenerator(RelaxInputsGenerator):
+    """Input generator for the `VASPRelaxWorkChain`."""
+
+    _default_protocol = 'efficiency'
+    _protocols = {'efficiency': {'description': ''}, 'precision': {'description': ''}}
```

Comment:
Even if you don't really implement any protocols yet, please already use the agreed upon terms. So just define a single one called `moderate` and set that also as the default. Fine for now if `get_builder` doesn't actually do anything with it

Reply:
Yes, this was just WIP after starting from the QE example. Thanks for the comment.